src/coreclr/tools/aot/ILCompiler.ReadyToRun/JitInterface/CorInfoImpl.ReadyToRun.cs

```diff
@@ -1337,6 +1346,9 @@ private void PublishEmptyCode()
         {
             _methodCodeNode.SetCode(new ObjectNode.ObjectData(Array.Empty<byte>(), null, 1, Array.Empty<ISymbolDefinitionNode>()));
             _methodCodeNode.InitializeFrameInfos(Array.Empty<FrameInfo>());
+#if READYTORUN
```

Comment:
There's no need to use <code>#if READYTORUN</code> in <code>CorInfoImpl.ReadyToRun.cs</code>, this source file is only compiled when <code>READYTORUN</code> is set.